runtime/src/main/java/io/quarkiverse/githubapp/runtime/config/ConfigChecker.java

```diff
@@ -0,0 +1,50 @@
+package io.quarkiverse.githubapp.runtime.config;
+
+import java.util.Set;
+import java.util.TreeSet;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import io.quarkus.runtime.LaunchMode;
+
+@Singleton
+public class ConfigChecker {
+
+    @Inject
+    GitHubAppRuntimeConfig gitHubAppRuntimeConfig;
+
+    @Inject
+    LaunchMode launchMode;
+
+    public void checkConfig() {
+        Set<String> missingPropertyKeys = new TreeSet<>();
+
+        if (gitHubAppRuntimeConfig.appId.isEmpty()) {
+            missingPropertyKeys.add("quarkus.github-app.app-id (QUARKUS_GITHUB_APP_APP_ID)");
+        }
+        if (gitHubAppRuntimeConfig.privateKey.isEmpty()) {
+            missingPropertyKeys.add("quarkus.github-app.private-key (QUARKUS_GITHUB_APP_PRIVATE_KEY)");
+        }
+        if (launchMode == LaunchMode.NORMAL && gitHubAppRuntimeConfig.webhookSecret.isEmpty()) {
+            missingPropertyKeys.add("quarkus.github-app.webhook-secret (QUARKUS_GITHUB_APP_WEBHOOK_SECRET)");
+        }
```

Comment:
I was building the set of missing property keys in the constructor on purpose, so that you could call `checkConfig` very often without any impact on memory allocation if the configuration is complete.